### ISSUE TITLE
There are two ENTERED_BY 

### ISSUE THREAD

**Issue by aazaff** [paleobot/pbot_client#15]
There appears to be two `ENTERED_BY` relations between user currano and organ leaf. Is this intentional or is this an example of why we need constraints?

**Comment by NoisyFlowers** [comment]
Ha, I'd say the latter!

**Comment by NoisyFlowers** [comment]
The annoying thing about this particular case is that I had already tried to avoid duplication of Organ nodes by using MERGE on both the node and the relationship. But the timestamp field in the relationship broke this. 

I'm not sure a constraint will help here. We want to allow only one ENTERED_BY relationship for a node. I'm not sure that is possible with a constraint.

Instead, I am changing the Organ declarations in OTUs.cypher from this:
```
MATCH
	(person:Person {given: "Ellen", surname: "Curranno"})
MERGE
    (organ:Organ { type: "Thing"})
    ON CREATE SET organ.organID = apoc.create.uuid()
MERGE
    (organ)-[:ENTERED_BY {timestamp: datetime()}]->(person);
```
To this:
```
MATCH
	(person:Person {given: "Ellen", surname: "Curranno"})
MERGE
    (organ:Organ { type: "Stem"})
    ON CREATE SET organ.organID = apoc.create.uuid()
MERGE
    (organ)-[entered_by:ENTERED_BY]->(person)
    ON CREATE SET entered_by.timestamp = datetime();
```

**Comment by NoisyFlowers** [comment]
A bit more streamlined:
```
MATCH
	(person:Person {given: "Ellen", surname: "Curranno"})
MERGE
    (organ:Organ { type: "Leaf"})-[entered_by:ENTERED_BY]->(person)
    ON CREATE SET 
        organ.organID = apoc.create.uuid(),
        entered_by.timestamp = datetime();
```

**Comment by NoisyFlowers** [comment]
This should be fixed with d37b4743b2b2122185443a5c7067ebe8df69943c

**Comment by aazaff** [comment]
If we only allow one entered-by per node, how will we track edits?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: NoisyFlowers ***@***.***>
Sent: Monday, March 29, 2021 2:09:37 PM
To: paleobot/pbot_client ***@***.***>
Cc: Zaffos, Andrew - (azaffos) ***@***.***>; Author ***@***.***>
Subject: [EXT]Re: [paleobot/pbot_client] There are two ENTERED_BY (#15)


External Email

This should be fixed with d37b474<https://github.com/paleobot/pbot_client/commit/d37b4743b2b2122185443a5c7067ebe8df69943c>

**Comment by NoisyFlowers** [comment]
This fix doesn't disallow multiple ENTERED_BY relationships. It just prevents them in this specific instance (creating an organ node as part of the process of creating an OTU. 

You can still use a CREATE instead of a MERGE and you'll get another relationship.

We'll have to figure out when we actually want this and when we don't.
